pkg/ottl/ottlfuncs/README.md

```diff
@@ -600,6 +600,31 @@ Examples:
 - `Duration("333ms")`
 - `Duration("1000000h")`
 
+
+### ElementizeAttributesXML
+
+`ElementizeAttributesXML(target, Optional[xpath])`
+
+The `ElementizeAttributesXML` Converter returns an edited version of an XML string where attributes are converted into child elements.
```

Comment:
I am worried about the discoverability of this function.  Is `Elementize` a well known term in XML communities? A more verbose, but specific, name would be helpful.

Reply:
"Element" is a well known term in XML, so I thought "Elementize" conveyed "to make elements", but it's definitely a contrived name. 

Maybe `ConvertAttributesToElementsXML`? I'm open to other suggestions.

Reply:
@TylerHelmuth I've updated the name to `ConvertAttributesToElementsXML`. If this looks good to you, I'll make a similar change to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35364